docs/release-notes/2.41.0.md

```diff
@@ -0,0 +1,58 @@
+
+# Percona Monitoring and Management 2.41.0
+
+
+| **Release date:** | Oct 05, 2023                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+<!---
+
+!!! caution alert alert-warning "Important/Caution"
+    Crucial points that need emphasis:
+
+    - Important: A significant point that deserves emphasis.
+    - Caution: Used to mean 'Continue with care'.
+
+--->
+
+## Release Highlights
+### Data collection is not can survive temporary Network problems between Database and PMM server 
+The PMM software is designed to monitor and troubleshoot, but there may be instances where it is unable to collect data due to connection issues. This is usually caused by a disruption in communication between the PMM server and the monitored databases. In the past, this resulted in a period of time where no data was collected. However, with the latest release, all types of data, including Service Metrics and Query Metrics, will be collected during such disruptions. This data will be stored and visible on the UI as soon as the connection is restored. This will enable users to investigate issues caused by server load or other problems that may have disrupted the data collection.
+
+
+### Simplified Reporting about Database problems to Percona
+Efficient resolution of a problem often relies on a well-written report detailing the issue. 
+To simplify the reporting process for our customers, we created an internal tool some time ago. We later added it to the PMM server, but its execution was complex, requiring users to install additional UI or use the terminal. 
+However, in this release, we have made the tool PMM Dump(https://docs.percona.com/pmm-dump-documentation/index.html) easily accessible for all PMM users through the PMM user interface. 
+Both Percona customers and community users can benefit from its use in collecting data for bug reports or internal usage. This marks the first step towards improving data gathering and triaging problems more efficiently.
+ 
+### PostgreSQL monitoring performance improvement
+At our company, performance is a crucial aspect of our work and we prioritize the principle of "Do no Harm." Recently, we received reports about excessive resource usage in monitoring PostgreSQL, particularly with regard to connections being used. In this latest release, we have made further improvements to optimize resource usage for monitoring, resulting in pmm using significantly fewer connections compared to previous versions and other resources.
```

Comment:
```suggestion
At our company, we prioritize the principle of "Do no Harm" and recognize that performance is a crucial aspect of our work. With this release, we have addressed a common problem for our PostgreSQL users regarding efficient resource usage by PMM. Specifically, we have decreased the number of used connections for monitoring, resulting in improved efficiency.
```

Reply:
moved doc to https://www.notion.so/percona/PMM-2-41-3015eb1f559242008595906b117dff5f